### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by Lihua93**
./robots/issue-creator/creator/creator_test.go: dont -> don't
./robots/issue-creator/test_owners.csv: commmand -> command

### CODE REVIEW

File: robots/issue-creator/test_owners.csv [hunk 1]
```diff
@@ -89,7 +89,7 @@ DisruptionController should create a PodDisruptionBudget,rkouj,0,scheduling
 DisruptionController should update PodDisruptionBudget status,rkouj,0,scheduling
 Docker Containers should be able to override the image's default arguments (docker cmd),maisem,0,node
 Docker Containers should be able to override the image's default command and arguments,maisem,0,node
-Docker Containers should be able to override the image's default commmand (docker entrypoint),maisem,0,node
```
Comment (rmmh) [inline robots/issue-creator/test_owners.csv]: This typo needs to be fixed in kubernetes/kubernetes first!

Comment (Lihua93) [reply robots/issue-creator/test_owners.csv]: Working on it.

Comment (Lihua93) [reply robots/issue-creator/test_owners.csv]: This typo has been fixed in kubernetes/kubernetes.
What should I do next?

**Pull request merged**